docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc

```diff
@@ -0,0 +1,92 @@
+
+== Before you start
+
+* Set up a Jira Data Center instance in the cloud or on-premise.
+* For Cloud and Hybrid customers, ensure that the Jira Data Center can connect to the Kobiton portal. The most straightforward way is to host Jira Data Center on a public server. If this is not an option, additional instructions will come soon.
+* For Standalone/On-prem customers, instructions to connect to Jira Data Center will come soon.
+* An admin account on Jira Data Center and an account with ADMIN predefined role in Kobiton.
+* Subscribe your Kobiton account to a plan with the Jira Integration item.
+
+== Create application links
+
+Log into Jira Data Center as an admin.
+
+Select the **gear (Setting)** icon, then **Applications**.
+
+image::integrations:integration-jira-enable-jira-dc-integration-gear-application.PNG[width=300,alt="Opening the Settings menu and going to Applications"]
+
+Enter your username and password again and choose **Confirm**.
+
+Select **Application links** under Integrations, then **Create link**.
+
+image::integrations:integration-jira-enable-jira-dc-integration-app-link-create-link.PNG[width=700,alt="Selecting Applications links under Integrations option and clicking the Create Link button"]
+
+Select **External Application** for Application type, then select **Incoming** for Direction.
+
+image::integrations:integration-jira-enable-jira-dc-integration-external-app-incoming.PNG[width=700,alt="In the Create link menu, selecting External application and incoming options, and then clicking continue"]
+
+In the Configure an incoming link page, input all the required fields below:
+
+* A name for the link.
+* For redirect url, input `https://api.kobiton.com/v1/jira-integration/callback` .
+* For permission, select **Write**.
+
+image::integrations:integration-jira-enable-jira-dc-integration-write-configure-incoming-link.PNG[width=700,alt="Filling in the incoming link detials: name, redirect url, and permissoin"]
+
+Select **Save** when done.
+
+The Credentials page displays.
+
+image::integrations:integration-jira-enable-jira-dc-integration-credentials.PNG[]
+
+[TIP]
+===============================
+You can always revisit the Credentials page by select the **more** icon of the link in the Application links list and choose **View credentials**.
+
+image::integrations:integration-jira-enable-jira-dc-integration-tip-application-links.PNG[width=600,alt="Choosing View credentials by clicking the more icon"]
+
+===============================
+
+Copy **Client ID** and **Client secret** and note them down.
+
+Select the **System** tab.
+
+Copy the value of **Base URL** and note it down
+
+image::integrations:integration-jira-enable-jira-dc-integration-base-url-administration.PNG[width=700,alt="Going to the systems tab and copying the Base URL to note it down or save it"]
+
+== Enable Jira Data Center integration
+
+Log into the Kobiton portal with an account with ADMIN predefined role.

```

Comment:
Log in to the Kobiton portal with an account with the ADMIN predefined role.

Reply:
into changed to in to